### PR TITLE
Add new version of conntrols file for SYS.1.6.A12 and SYS.1.6.A13- no…

### PR DESCRIPTION
**Pull request by lichtblaugue**
…w based on master branch

#### Description:

-  Purge Request for SYS.1.6.A12 and SYS.1.6.A13 

#### Rationale:

-  Requested BSI Profile from our customers

### CODE REVIEW

File: controls/bsi_sys_1_6.yml [hunk 1]
```diff
@@ -194,17 +194,27 @@ controls:
...
       eventually deployed to a productive environment SHOULD be adequately documented.
-      Images used SHOULD have metadata that makes their function and history traceable. Digital
+      (3) Images used SHOULD have metadata that makes their function and history traceable. (4) Digital
       signatures SHOULD secure each image against modification.
     notes: >-
-      ToDo
-    status: manual
-    #rules:
-
+      Section 1: This requirement must be implemented organizationally.
```
Comment (benruland) [inline controls/bsi_sys_1_6.yml:204]: How about adding these rules?
- `approved_registries`
- `ocp_allowed_registries`
- `ocp_allowed_registries_for_import`

Comment (sluetze) [reply controls/bsi_sys_1_6.yml:204]: approved_registries is a AWS EKS Rule, I would not use it.

Comment (lichtblaugue) [reply controls/bsi_sys_1_6.yml:204]: I will add the other two rules

Comment (lichtblaugue) [reply controls/bsi_sys_1_6.yml]: New version pushed

**Review by yuumasato — APPROVED** [review]
/lgtm

**Comment by yuumasato** [comment]
Merging, :P

**Pull request merged**